### ISSUE TITLE
Sill doesnt stay enabled

### ISSUE THREAD

**Issue by Vansmak** [aldadic/home-assistant-on-echo-show#1]
the skill shows in alexa under dev but not enabled, i enable it but every time i go back to it it is disabled again.  I have tried to trigger the skill from homeassistant and nothing happens.  i did insert the skills code correctly.

**Comment by Vansmak** [comment]
it may have been because I was logged into dev console on 2 devices at dame time. I've been away this week so haven't played with it since

**Comment by aldadic** [comment]
That might actually be the issue. Please post an update once you tried it again, I'm really interested to know if this solves the problem.

**Comment by Vansmak** [comment]
yeah that was the issue as far as that goes. now this bigger issue is getting the automation to fire every 5 minutes from HA. after awhile the echo home creek returns and stay on indefinitely
